src/python/tests/core/bot/tasks/corpus_pruning_task_test.py

```diff
@@ -319,16 +319,15 @@ def setUp(self):
     environment.set_value('JOB_NAME', 'libfuzzer_asan_job')
 
     helpers.patch(self, [
-        'bot.fuzzers.engine.get',
-        'bot.tasks.setup.get_fuzzer_directory',
+        'bot.fuzzers.engine.get', 'bot.tasks.setup.get_fuzzer_directory',
```

Comment:
nit: did auto-format do that?

Reply:
I believe so, fixed.

Reply:
Turns out the linter wants these two to be on the same line. Changed it back